platform/android/export/export.cpp

```diff
@@ -683,6 +683,22 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return enabled_plugins;
 	}
 
+    static Error store_in_gradle_project(const String &p_path, const Vector<uint8_t> &p_data, int compression_method = Z_DEFLATED) {
```

Comment:
The indent of all of your new functions doesn't match existing code - might need to update your IDE style settings